src/main/java/hudson/plugins/gradle/injection/InjectionConfig.java

```diff
@@ -223,6 +240,24 @@ public void setMavenInjectionDisabledNodes(List<NodeLabelItem> mavenInjectionDis
             mavenInjectionDisabledNodes == null ? null : ImmutableList.copyOf(mavenInjectionDisabledNodes);
     }
 
+    @DataBoundSetter
+    public void setInjectionVcsRepositoryPatterns(String injectionVcsRepositoryPatterns) {
+        this.injectionVcsRepositoryPatterns = Util.fixEmptyAndTrim(injectionVcsRepositoryPatterns);
+
+        this.parsedInjectionVcsRepositoryPatterns = this.injectionVcsRepositoryPatterns == null
+                ? ImmutableList.of()
+                : ImmutableList.copyOf(Arrays.stream(this.injectionVcsRepositoryPatterns.split(",")).map(String::trim).collect(Collectors.toList()));
```

Comment:
We can also do this in a `getParsedInjectionVcsRepositoryPatterns() ` getter every time it's invoked, but since it's changes only via configuration change in UI, I think it's ok to have it precomputed